### DISCUSSION TITLE
Oracle (Innocent)

### DISCUSSION THREAD

**Discussion by Alf21** [Alf21/TTT2#753] [Role Ideas]
- he plays in the innocent team
- he get’s a DNA like notification when a player is killed for a short time (like a wallhack). It also shows him the role of the killed player for the short time
- his role is hidden like every other innocent role

Idea by: @TimGoll

**Comment by mexikoedi** [comment]
There is a similar role but not exactly how it is described here: https://steamcommunity.com/sharedfiles/filedetails/?id=2620857098
